phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widgets/lists/icons/IconsListElement.java

```diff
@@ -80,13 +81,14 @@ public IconsListElement(E item, URI miniatureUri, String title, boolean enabled)
             this.setClip(clip);
         });
 
-        final Label label = new Label(title);
-        label.getStyleClass().add("miniatureText");
+        final Region miniature = new Region();
```

Comment:
I don't want to make the URIs theme dependent. I just want to get rid of a class which is only used to store three static URIs.